### ISSUE TITLE
Using Pytorch

### ISSUE THREAD

**Issue by chho-work** [akTwelve/cocosynth#14]
Thanks for the code!
In your Udemy course, do you use Keras(TF) or Pytorch?  I would like to use Detectron2(Pytorch).  Does it work as well or it doesn't really matter which framework I use?  Thanks!

**Comment by akTwelve** [comment]
It doesn’t really matter since the majority of the course is focused on creating the dataset. The Matterport Mask R-CNN algorithm uses keras, but that part is just a bonus and probably isn’t the most practical tool for many applications anyway.

**Comment by chho-work** [comment]
Noted! Thanks!
